bin/git-authors

```diff
@@ -3,7 +3,6 @@
 LIST=false
 NO_EMAIL=false
 FILE=""
-EDITOR=$(git var GIT_EDITOR)
```

Comment:
We should update docs for removing this feature?

Reply:
Removed and update.